=== modified file 'kicad/CMakeLists.txt'
--- kicad/CMakeLists.txt        2014-10-17 17:45:33 +0000
+++ kicad/CMakeLists.txt	2014-11-06 02:44:06 +0000
@@ -105,6 +105,10 @@

         # do all the work
         include( BundleUtilities )
+
+        # If `BU_CHMOD_BUNDLE_ITEMS` is not set, `install_name_tool` will fail to re-write some loader paths due to lack of writable permissions if the build dependencies were installed by brew (or didn't have writable permissions)
+        set(BU_CHMOD_BUNDLE_ITEMS ON)
+
         fixup_bundle( ${OSX_BUNDLE_INSTALL_BIN_DIR}/kicad
             \"\${BUNDLE_FIX_LIBS}\"
             \"\"
