--- src/osx/cocoa/filedlg.mm  2014-06-24 18:43:01.000000000 -0300
+++ src/osx/cocoa/filedlg.mm  2015-01-03 17:26:48.000000000 -0200
@@ -679,6 +679,12 @@
         {
             panel = oPanel;
             result = wxID_OK;
+
+            if (m_filterChoice)
+            {
+                m_filterIndex = m_filterChoice->GetSelection();
+            }
+
             NSArray* filenames = [oPanel filenames];
             for ( size_t i = 0 ; i < [filenames count] ; ++ i )
             {
