diff --git a/common/pgm_base.cpp b/common/pgm_base.cpp
index b11d7ff..f5b3a93 100644
--- a/common/pgm_base.cpp
+++ b/common/pgm_base.cpp
@@ -37,6 +37,7 @@
 #include <wx/filename.h>
 #include <wx/snglinst.h>
 #include <wx/stdpaths.h>
+#include <wx/sysopt.h>
 
 #include <pgm_base.h>
 #include <wxstruct.h>
@@ -347,6 +348,9 @@ const wxString& PGM_BASE::GetEditorName()
 
 bool PGM_BASE::initPgm()
 {
+    // ask OS X to show filte filters on Open dialog for plugin disambiguation.
+    wxSystemOptions::SetOption(wxOSX_FILEDIALOG_ALWAYS_SHOW_TYPES, 1);
+
     wxFileName pgm_name( App().argv[0] );
 
     wxConfigBase::DontCreateOnDemand();
